vyper/functions/functions.py

```diff
@@ -632,9 +632,14 @@ def minmax(expr, args, kwargs, context, is_min):
         comparator = 'gt' if is_min else 'lt'
     else:
         comparator = 'sgt' if is_min else 'slt'
-    if left.typ.typ == right.typ.typ:
+    if left.typ.typ == right.typ.typ or \
+            (all([typ in ('int128', 'uint256') for typ in (left.typ.typ, right.typ.typ)]) and
```

Comment:
There is a much nicer pattern one can use for this type of check ;)

```python
or {left.typ.typ, right.typ.typ} == {'int128', 'uint256'}
```

Reply:
This would be `False` if the inputs are `{uint256, uint256}`! :)

Looking at this as an option more closely led me to `{left.typ.typ, right.typ.typ} - {'int128', 'uint256'} == set()` which would work as expected. However, after considering your other comment this whole block might change and this mightn't be required!

Reply:
Right but the first part of or statement would catch that case.

Reply:
Haha, true!